.travis.yml

```diff
@@ -0,0 +1,21 @@
+#.travis.yml
+sudo: required
+dist: trusty
+language: node_js
+node_js:
+  - '8'
+  - '7'
+before_install:
+  - rm -rf node_modules
+  - npm install -g truffle
+  - npm install -g ganache-cli
+install:
+  - npm install
+#IF on contract branch - start testRPC || ELSE - move on  
+before_script: 
+  - cd $TRAVIS_BUILD_DIR
+  - "chmod +x ./BranchPreScriptCheck.sh && ./BranchPreScriptCheck.sh"
```

Comment:
I wonder if scripts can be run without `chmod`, just `"sudo ./BranchPreScriptCheck.sh"` or `"sudo $(which bash) ./BranchPreScriptCheck.sh"`